docs/R-notebooks/CorrelationShinyApp/ui.R

```diff
@@ -0,0 +1,177 @@
+                                        code("date"), "will be considered the last available signal date ", code("+ lag"), ". ",
+                                        width = 12)
+                                      )
+                             ),
+                            
+                             tabPanel("Signals from API",
+                                      
+                                      # Sidebar with a slider input for number of bins
+                                      sidebarLayout(
+                                        sidebarPanel(
+                                          radioGroupButtons(
+                                            inputId = "geo_type",
+                                            label = h4("Geographical granularity"), 
+                                            choices = c("county", "msa", "state"), 
+                                            justified = TRUE,
+                                            checkIcon = list(
+                                              yes = icon("ok", 
+                                                         lib = "glyphicon"))
+                                          ), 
+                                          dateInput("date_var", label = h4("Date for response"), value = "2020-04-17"), 
+                                          numericInput("lag", label = h4("Lag of days for correlation"), value = "0"),
+                                          numericInput("window", label = h4("Window (number of days to average)"), value = "7"),
+                                          pickerInput(
+                                            inputId = "response",
+                                            label = h4("Response variable"), 
+                                            choices = list(
+                                              common = likely_responses,
+                                              other = signal_meta), 
+                                            selected = c("jhu-csse: confirmed_incidence_prop")
+                                          ), 
+                                          multiInput(
+                                            inputId = "signals_to_plot",
+                                            label = h4("Signals to correlate with response"), 
+                                            choices = signal_meta, 
+                                            selected = c("doctor-visits: smoothed_cli",
+                                                         "fb-survey: smoothed_cli",
+                                                         "fb-survey: smoothed_hh_cmnty_cli",
+                                                         "google-survey: smoothed_cli", 
+                                                         "ght: smoothed_search", 
+                                                         "indicator-combination: nmf_day_doc_fbs_ght")
```

Comment:
This combination signal is no longer produced. You probably want `nmf_day_doc_fbc_fbs_ght`.